openlibrary/plugins/worksearch/code.py

```diff
@@ -1008,7 +1009,6 @@ def random_author_search(limit=10):
 
     return json.dumps(search_results['response'])
 
-
 def rewrite_list_editions_query(q, page, offset, limit):
```

Comment:
XXX `rewrite_list_editions_query` needs ctx default (see plugins/openlibrary/home.py) to avoid ctx.env miss when results are cached